components/var_dumper/introduction.rst

```diff
@@ -73,6 +73,26 @@ current PHP SAPI:
     #. From time to time, run ``composer global update`` to have the latest
        bug fixes.
 
+Using VarDumper in your PHPUnit test suite
+------------------------------------------
+
+Since Symfony 2.7, the VarDumper component provides
```

Comment:
you should add a versionadded for this new feature, like:

```rst
.. versionadded:: 2.7

    The :class:`Symfony\\Component\\VarDumper\\Test\\VarDumperTestTrait` was
    introduced in Symfony 2.7.
```

Then you should just write the first paragraph as if the feature always existed: "The VarDumper component provides a trait that can help writing some of your tests for PHPunit."